src/t8_schemes/t8_default/t8_default_hex/t8_dhex.h

```diff
@@ -35,6 +35,9 @@
 /** The number of faces of a hex. */
 #define T8_DHEX_FACES 6
 
+/** The number of vertices of a hex. */
+#define T8_DHEX_VERTICES 8
```

Comment:
I used that macro in a test i moved to another branch for a latter PR. I would still leave it in here, because it might be useful anyways.